src/solvers/ChambollePock.jl

```diff
@@ -47,9 +47,9 @@ For more details on the algorithm, see[^BergmannHerzogSilvaLouzeiroTenbrinckVida
 * `stopping_criterion` – (`stopAtIteration(100)`) a [`StoppingCriterion`](@ref)
 * `update_primal_base` – (`missing`) function to update `m` (identity by default/missing)
 * `update_dual_base` – (`missing`) function to update `n` (identity by default/missing)
-* `retraction_method` – (`ExponentialRetraction()`) the rectraction to use
-* `inverse_retraction_method` - (`LogarithmicInverseRetraction()`) an inverse retraction to use.
-* `vector_transport_method` - (`ParallelTransport()`) a vector transport to use
+* `retraction_method` – (`default_retraction_method(M)`) the rectraction to use
+* `inverse_retraction_method` - (`default_inverse_retraction_method(N)`) an inverse retraction to use.
```

Comment:
```suggestion
* `inverse_retraction_method` - (`default_inverse_retraction_method(M)`) an inverse retraction to use.
```

Are you sure about the `N` here? If I remember correctly, it might even be that both the retraction method and the inverse retraction method are used on _both_ manifolds. So if, it should be consistent and both should use the default from one manifold.

Reply:
I had never heard of Chambolle-Pock prior to this PR. The documentation suggests that the logarithmic map is evaluated at Λ
(p), which is a point on N and that the exponential map and parallel transport are evaluated on M.

Reply:
Oh, never mind, the paper introducing this algorithm was published only this January; and usually the name Chambolle-Pock is mainly known to the imaging community (other names are along the lines of primal dual hybrid gradient or PDHG, but since we had eRCP and lRCP variants of CP, we took the shorter abbreviations to not write eRPDHG).

As you can see https://manoptjl.org/stable/solvers/ChambollePock.html there is a logarithmic map on N in step 1 and a logarithmic map on M in step 4. The exponential map is only used on M though.

I think the paper also only uses exp/log, but sure for implementation one can easily go for retractions.

Reply:
In that case, it would make sense to modify both the solver and `ChambollePockOptions` to use a different inverse retraction for M and N. That would be for a different PR though.

Reply:
Sure, that sounds reasonable but you are right that is for another PR :)